### PR TITLE
Add initial OCP3 structure, C2S Profiles, and CPE content

### PR DESCRIPTION
**Pull request by redhatrises**
#### Description:

- Add initial OCP3 structure, C2S Profiles, and CPE content

#### Rationale:

- Currently, there is no OCP content at all.

### CODE REVIEW

**Comment by mpreisler** [comment]
The CMakeListst.txt does nothing and it all looks like a placeholder

**Comment by redhatrises** [comment]
That's correct. Would like to get this in so people can start helping contributing towards it.

**Comment by jan-cerny** [comment]
@yuumasato Do you think we can merge it before 0.1.36 (which is coming soon)? I would merge it. It isn't mentioned in root CMakeLists so I think it shouldn't affect the built content.

**Comment by yuumasato** [comment]
I'm okay with merging it.
@redhatrises To make it easier for people who want to contribute to build it, maybe add the product to `CMakeLists.txt` but leave it disabled by default?

**Comment by redhatrises** [comment]
> I'm okay with merging it.
@redhatrises To make it easier for people who want to contribute to build it, maybe add the product to CMakeLists.txt but leave it disabled by default?

@yuumasato done

File: CMakeLists.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ option(SSG_PRODUCT_FIREFOX "If enabled, the Firefox SCAP content will be built"
 option(SSG_PRODUCT_JBOSS_EAP6 "If enabled, the JBoss EAP6 SCAP content will be built" TRUE)
 option(SSG_PRODUCT_JBOSS_FUSE6 "If enabled, the JBoss Fuse6 SCAP content will be built" TRUE)
 option(SSG_PRODUCT_JRE "If enabled, the JRE SCAP content will be built" TRUE)
+option(SSG_PRODUCT_OCP3 "If enabled, the RHOCP3 SCAP content will be built" TRUE)
```
Comment (yuumasato) [inline CMakeLists.txt]: Why not set this to FALSE?

Comment (redhatrises) [reply CMakeLists.txt]: @yuumasato this was to stay consistent with how `RHEVM3` was disabled.

Comment (yuumasato) [reply CMakeLists.txt]: Oh, ok, it is not a big deal.

**Pull request merged**